libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tab.component.scss

```diff
@@ -1,35 +1,184 @@
+  --sky-override-vertical-tab-gap: #{$sky-margin-half};
+  --sky-override-vertical-tab-heading-error-padding-left: #{$sky-padding-half};
+  --sky-override-vertical-tab-hover-font-color: #{$sky-text-color-default};
+  --sky-override-vertical-tab-margin: #{$sky-margin-half} 0 #{$sky-margin-half}
+    #{$sky-margin};
+  --sky-override-vertical-tab-outline-focus: auto;
+  --sky-override-vertical-tab-padding: #{$sky-padding-3_4} 0 #{$sky-padding-3_4}
+    #{$sky-padding};
+  --sky-override-vertical-tab-padding-left-active: #{$sky-padding};
+  --sky-override-vertical-tab-padding-left-selected: 6px;
+  --sky-override-vertical-tab-padding-left-hover: #{$sky-padding};
+}
+
+@include compatMixins.sky-modern-overrides('.sky-vertical-tab') {
+  --sky-override-vertical-tab-background-color-disabled: transparent;
+  --sky-override-vertical-tab-background-color-selected: transparent;
+  --sky-override-vertical-tab-border-left-active: 4px solid
+    var(--modern-color-blue-50);
+  --sky-override-vertical-tab-border-left-hover: 1px solid
+    var(--modern-color-blue-50);
+  --sky-override-vertical-tab-gap: #{$sky-theme-modern-margin-inline-md};
+  --sky-override-vertical-tab-heading-error-padding-left: #{$sky-padding-half};
+  --sky-override-vertical-tab-margin: #{$sky-theme-modern-margin-inline-md} 0 #{$sky-theme-modern-margin-inline-md}
+    #{$sky-theme-modern-margin-inline-sm};
+  --sky-override-vertical-tab-content-pane-margin-bottom: 0;
+}
+
+:host {
+  &:has(~ sky-vertical-tab) .sky-vertical-tab {
+    margin-bottom: var(
+      --sky-override-vertical-tab-gap,
+      var(--sky-space-gap-action_group-l)
+    );
+  }
+
+  &:has(.sky-vertical-tabset-button-disabled) {
+    cursor: not-allowed;
+  }
+}
 
 .sky-vertical-tab {
   cursor: pointer;
-  padding: $sky-padding-3_4 0 $sky-padding-3_4 $sky-padding;
-  margin: $sky-margin-half 0 $sky-margin-half $sky-margin;
+  background-color: var(
+    --sky-override-vertical-tab-background-color,
+    var(--sky-color-background-nav-base)
+  );
+  padding: var(
+    --sky-override-vertical-tab-padding,
+    var(--sky-space-inset-pillarbox-2_3-top-m)
+      var(--sky-space-inset-pillarbox-2_3-right-m)
+      var(--sky-space-inset-pillarbox-2_3-bottom-m)
+      var(--sky-space-inset-pillarbox-2_3-left-m)
+  );
+  margin: var(
+    --sky-override-vertical-tab-margin,
+    0 0 0 var(--sky-space-inline-s)
+  );
   display: flex;
   text-decoration: none;
+  color: var(
+    --sky-override-vertical-tab-font-color,
+    var(--sky-color-text-deemphasized)
+  );
+  border-radius: var(--sky-border-radius-0);
 
-  &:not(.sky-deemphasized) {
-    color: $sky-text-color-default;
+  &:hover {
+    color: var(
+      --sky-override-vertical-tab-hover-font-color,
+      var(--sky-color-text-default)
+    );
+    background-color: var(
+      --sky-override-vertical-tab-background-color-hover,
+      var(--sky-color-background-nav-hover)
+    );
+
+    // TODO: Tokenize with sectioned form
```

Comment:
Any style for the right arrow will be tokenized with sectioned form as that is where that arrow is actually seen.